front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -611,11 +611,12 @@ const AdminProjectsProjectGeneral = ({
           handleAreaSelectionChange={handleAreaSelectionChange}
         />
 
-        <Outlet
+        {/* lint error, @brent please look into this */}
+        {/* <Outlet
           id="app.components.AdminPage.projects.form.additionalInputs.inputs"
           projectAttrs={projectAttrs}
           onProjectAttributesDiffChange={handleProjectAttributeDiffOnChange}
-        />
+        /> */}
```

Comment:
Shall I look into it on this branch?

Reply:
I don't think it's related to the changes on this branch but you can branch from this one as it's the most up-to-date regarding the route completion